src/libutils/oglapphelpers/CMakeLists.txt

```diff
@@ -39,13 +38,59 @@ target_include_directories(oglapphelpers
         ${GLUT_INCLUDE_DIR}
 )
 
-target_link_libraries(oglapphelpers
-    PRIVATE
-        OpenColorIO
-        ${OPENGL_LIBRARIES}
-        ${GLEW_LIBRARIES}
-        ${GLUT_LIBRARIES}
-)
+
+if(${OCIO_IS_NVIDIA})
+    if(${OCIO_EGL_HEADLESS})
+        target_include_directories(oglapphelpers
+            PRIVATE
+                ${OPENGL_EGL_INCLUDE_DIRS}
+        )
+        target_link_libraries(oglapphelpers
+            PRIVATE
+                OpenColorIO
+                OpenGL::OpenGL
+                OpenGL::GLU
+                ${GLEW_LIBRARIES}
+                ${GLUT_LIBRARIES}
```

Comment:
`GLUT` is then not needed!

Reply:
Hi, `GLUT` is still used in `ScreenApp`.